### DISCUSSION TITLE
Cannot resolve the opendcs_cs:opendcs_cs:7.0.1.5 dependency during build

### DISCUSSION THREAD

**Discussion by DipeshPatel-USDA** [opendcs/rest_api#17] [Q&A]
I am trying to build the rest_api project, and I'm unable to resolve the opendcs_cs dependency.  Where can I find that dependency?

**Comment by MikeNeilson** [comment]
Was wondering that myself when I saw it. I haven't had a chance yet but you can probably just change it to 

```
<dependency>
			<groupId>org.opendcs</groupId>
			<artifactId>opendcs</artifactId>
			<version>7.0.8</version>
</dependency>
```

**Reply by DipeshPatel-USDA** [reply]
Ok, thanks.  That's what I did yesterday to get it to work, but I had to exclude the transitive dependencies because I couldn't access some internal jars from USACE.  I did this, and I was able to build:

```
<dependency>
	<groupId>org.opendcs</groupId>
	<artifactId>opendcs</artifactId>
	<version>7.0.8</version>
	<exclusions>
		<exclusion>	
			<groupId>*</groupId>
			<artifactId>*</artifactId>
		</exclusion>
	</exclusions>
</dependency>
```
